modules/network-observability-netobserv-cli-reference.adoc

```diff
@@ -13,183 +15,96 @@ $ oc netobserv [<command>] [<feature_option>] [<command_options>] <1>
 ----
 <1> Feature options can only be used with the `oc netobserv flows` command. They cannot be used with the `oc netobserv packets` command.
 
+== Basic commands
 [cols="3a,8a",options="header"]
 .Basic commands
 |===
-|Command| Description
-
+| Command | Description
 | `flows`
-| Capture flows information. For subcommands, see the "Flow capture subcommands" table. 
-
+| Capture flows information. For subcommands, see the "Flow capture subcommands" table.
 | `packets`
-| Capture packets from a specific protocol or port pair, such as `netobserv packets --filter=tcp,80`. For more information about packet capture, see the "Packet capture subcommand" table.
-
+| Capture packets from a specific protocol or port pair, such as netobserv packets --filter=tcp,80. For more information about packet capture, see the "Packet capture subcommand" table.
```

Comment:
No need to pass `--filter=tcp,80` for packets. That is removed from this release

Reply:
I'm updating this in cli repo :+1: